src/index.html

```diff
@@ -1,15 +1,117 @@
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="icon" href="https://i.imgur.com/9IvZHDK.png">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="POST" onsubmit="onSubmit()">
+      <fieldset class="contextBlock">
+        <legend>Personal information</legend>
+        <label for="surname" class="inputBlock">
+          Surname:
+          <input type="text" name="gender"
+                 id="surname">
+        </label>
+        <label for="name" class="inputBlock">
+          Name:
+          <input type="text" name="gender"
+                 id="name">
+        </label>
+        <label for="age" class="inputBlock">
+          How old are You?
+          <input type="number" id="age"
+                 value="12" maxlength="3"
+                 min="6" max="110">
+        </label>
+        <label for="birthday" class="inputBlock">
+          Full date of birth:
+          <input type="date" id="birthday">
+        </label>
+        <label for="readTerms" class="inputBlock">
+          I accept the term of the agreement
+          <input type="checkbox" id="readTerms"
+                 required>
+        </label>
+      </fieldset>
+
+      <fieldset class="contextBlock">
+        <legend>Registration</legend>
+        <label for="email" class="inputBlock">
+          E-mail:
+          <input type="email" id="email"
+                 placeholder="email@example.com">
+        </label>
+
+        <label for="pass" class="inputBlock">
+          Password:
+          <input type="password" id="pass"
+          minlength="7">
+        </label>
+      </fieldset>
+
+      <fieldset class="contextBlock">
+        <legend>An interesting fact about yoy!</legend>
+        <label for="catYes" class="inputBlock">
+          Do yoy love cats?
+          <input type="radio" name="catsAnswer"
+                 id="catYes">
+          <span>Yes</span>
+          <input type="radio" name="catsAnswer"
+                 id="catNo">
+          <span>No</span>
+        </label>
+        <label for="favColor" class="inputBlock">
+          What is your favorite color?
+          <input type="color" id="favColor">
+        </label>
+        <label for="goToBed" class="inputBlock">
+          What time do you go to bed?
+          <input type="time" id="goToBed"
+                 step="1">
```

Comment:
```suggestion
          <input 
            type="time" 
            id="goToBed"
            step="1"
           >
```
отредактируй все места где много атрибутов подобным образом

Reply:
Я тільки там де більше 2-х атрибутів поправлю?

Reply:
done